app/controllers/dashboard_controller.rb

```diff
@@ -17,15 +23,17 @@ def exception
   end
 
   def search_queries
-    @queries = SearchQuery.where(parent_id: nil).order(id: :desc).includes(:user).limit(100)
+    @queries = SearchQuery.where(query_type: 'search').where(parent_id: nil).order(id: :desc).includes(:user).limit(100).decorate
   end
 
-
+  def list_queries
+    parent_ids = SearchQuery.where.not(parent_id: nil).pluck(:parent_id).uniq
+    @queries = SearchQuery.where(query_type: 'list').where(id: parent_ids).order(id: :desc).includes(:user).limit(100).decorate
```

Comment:
I know I did this, but it's actually wrong - we still need ```where(parent_id: nil)``` in addition to all the ohter where clauses, otherwise we will end up with the parent in a grandparent - parent - child situation